pom.xml

```diff
@@ -550,6 +553,16 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.curator</groupId>
+        <artifactId>curator-client</artifactId>
+        <version>${curator-version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.curator</groupId>
+        <artifactId>curator-recipes</artifactId>
+        <version>${curator-version}</version>
+      </dependency>
```

Comment:
I don't think these should be included in the root POM file since these are specific to the ZKProvider project/module

Reply:
Thanks, moved